### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    type = "select"
+    choices = ["VIEWER", "REPORTER", "UPDATER", "DEVELOPER", "MANAGER"]
+    default = "REPORTER"
+
+[resources]
+
+    [resources.sources]
+        [resources.sources.main]
+        url = "https://sourceforge.net/projects/mantisbt/files/mantis-stable/2.25.8/mantisbt-2.25.8.zip/download"
+        sha256 = "c8bd4b585183cb42cdd9fbb00c4e5d5945e01533882d7e534b33dc45025752ff"
```
Comment (alexAubin) [inline manifest.toml]:
```suggestion
        url = "https://github.com/mantisbt/mantisbt/archive/refs/tags/release-2.25.8.tar.gz"
        sha256 = "8d96bf93a283267fc088229661432831829d18b9fc1d20bff35df64be69ea27d"
```

zzz sourceforge download urls are completely stupid

**Comment by alexAubin** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Pull request closed without merging**